core/src/main/java/com/novoda/noplayer/exoplayer/playererror/ExoPlayerErrorFactory.java

```diff
@@ -1,27 +1,21 @@
-package com.novoda.noplayer.exoplayer.forwarder;
+package com.novoda.noplayer.exoplayer.playererror;
 
 import android.media.MediaCodec;
 
 import com.google.android.exoplayer2.ParserException;
 import com.google.android.exoplayer2.upstream.HttpDataSource;
 import com.novoda.noplayer.Player;
 import com.novoda.noplayer.drm.DrmRequestException;
-import com.novoda.noplayer.exoplayer.playererror.ConnectivityError;
-import com.novoda.noplayer.exoplayer.playererror.DrmDecryptionError;
-import com.novoda.noplayer.exoplayer.playererror.DrmRequestError;
-import com.novoda.noplayer.exoplayer.playererror.InvalidResponseCodeError;
-import com.novoda.noplayer.exoplayer.playererror.MalformedContentError;
-import com.novoda.noplayer.exoplayer.playererror.UnknownError;
 
 import java.io.IOException;
 
-final class ExoPlayerErrorFactory {
+public final class ExoPlayerErrorFactory {
```

Comment:
If this is public a client can do `ExoPlayerErrorFactory.errorFor(...)`. Is this part of the public API?